src/workflow/workflow.js

```diff
@@ -548,20 +550,43 @@ const setInstanceMetadata = async (wfConfig) => {
 };
 
 const sendInitialFunds = async (wfConfig) => {
-  // 7-fund accounts with some initial funds
-  const amount = wfConfig?.instance?.initialFund;
-
-  // if no initialFund is set or initialFund is set to zero, skip this step.
-  if (!amount) {
-    console.log(
-      'no initialFunds was set in workflow. skipping sendInitialFunds!'
-    );
-    return;
-  }
+  // 7 - fund accounts with some initial funds
+  let initialFund = wfConfig?.instance?.initialFund;
 
   const context = getContext();
-  const { startRecordNo, endRecordNo } = context.data;
   const { dryRun } = context;
+  const { startRecordNo, endRecordNo } = context.data;
+  const { api, signingPair } = context.network;
+
+  if (!initialFund.match(initialFundPattern)) {
+    const { calcInitialFund } = (await inqAsk([
+      {
+        type: 'confirm',
+        name: 'calcInitialFund',
+        message: `Would you like to calculate and set initialFund?`,
+        default: false,
+      },
+    ])) || { calcInitialFund: false };
+
+    if (calcInitialFund) {
+      const { id: collectionId, startInstanceId: itemId } = context.class;
+      const [destinationAddressColumn] = context.data.getColumns([
+        columnTitles.address,
+      ]);
+      const destinationAddress = destinationAddressColumn.records[0];
+      const { existentialDeposit } = api.consts.balances;
+
+      const info = await api.tx.uniques
+        .transfer(collectionId, itemId || 0, destinationAddress)
+        .paymentInfo(signingPair.address);
+
+      const fee = info.partialFee.mul(new BN(13)).div(new BN(10));
+      initialFund = existentialDeposit.add(fee);
```

Comment:
For the campaigns that we have run so far, we set this to 2*fee, just in case they might want to transfer it to another account after they claim. I'd say tx fees are very low so 2x makes sense. also helps to be safe in case tx fees increases by the time that they claim.